meteor/sofie-server/data-access/repositories/mongo/mongo-rundown-repository.ts

```diff
@@ -69,4 +71,16 @@ export class MongoRundownRepository extends BaseMongoRepository implements Rundo
 			)
 		}
 	}
+	/* Todo: Decide if a similar assertions should be performed for other entities.
+	 *		 If other entities should also assert similarly, then decide if this should be moved to 'BaseMongoRepository'.
+	 *		 If moved, should also be renamed to e.g 'assertEntityExist(...)'
+	 */
```

Comment:
Do you plan on doing this TODO in your task?
If yes, then you should go do it.
If no, it should be deleted.

If you are in doubt whether or not should do it, then you should ask :) 

Just leaving a TODO like this is how codebases get full on TODOs that will never happen.

Reply:
Todo comment removed, during our chat.